lib/sidekiq.rb

```diff
@@ -8,7 +8,6 @@
 require "sidekiq/client"
 require "sidekiq/transaction_aware_client"
 require "sidekiq/job"
-require "sidekiq/worker_compatibility_alias"
```

Comment:
Everything else is fine but I don’t want to go this far. S::W is not officially deprecated, just no longer recommended.

Reply:
Restored.  

I fixed the reference in more places.